### ISSUE TITLE
unit tests should all use standardized fixtures

### ISSUE THREAD

**Issue by alexkjames** [LinkedEarth/Pyleoclim_util#503]
Currently our unit tests use a mixture of bespoke testing objects and well defined pytest fixtures. We should move to use pytest fixtures across the board (as much as is reasonable anyway)

- [ ] coherence
- [ ] correns
- [ ] ensembleseries
- [ ] geoseries
- [ ] multiplegeoseries
- [ ] multiplepsd
- [ ] multipleseries
- [ ] multivariatedecomp
- [ ] psd
- [ ] resolution
- [ ] scalogram
- [ ] series
- [ ] ssares
- [ ] tsbase
- [ ] tsutils
- [ ]

**Comment by CommonClimate** [comment]
Yes! Let me add that most of our tests use series that are on the long side. We should endeavor to use as few data points as possible to make the tests faster, thereby accelerating our unit test cycles.

**Comment by CommonClimate** [comment]
I'm going to add as a subtask that every time we create a Series fixture, we set `auto_time_params` to `True` to avoid the annoying warning... Those really add up and flood the output of the CI tests.

**Comment by alexkjames** [comment]
Addressed by #647
